manifests/install.pp

```diff
@@ -31,6 +31,9 @@
         }
       }
 
+      'Debian':{
```

Comment:
@wolfaba maybe we should add a
`include java`

Reply:
@johnlawerance oh sorry, I have missed, this is only for Java and only if `install_java==true`.

@aba-rechsteiner OK, for Debian block there should be any `include java`. But I would like install java myself, therefore I don't need it in fact. We can disable `install_java` in our installation, becase we install java from our profile module.

AFAIK Jira and Confluence modules neither install java themself, so for me it's OK, if crucible module will not install java. But I understand for someone can be useful to install java for crucible in one config.

Reply:
Okay, then I'll close this PR